ios/Classes/SwiftAdmobFlutterPlugin.swift

```diff
@@ -38,7 +38,6 @@ public class SwiftAdmobFlutterPlugin: NSObject, FlutterPlugin {
     case "initialize":
         // https://developers.google.com/admob/ios/test-ads#enable_test_devices
         // **iOS simulators are automatically configured as test devices.**
-        // GADMobileAds.sharedInstance().requestConfiguration.testDeviceIdentifiers = ["You're physical iDevice Id"]
```

Comment:
When this plugin is initialized I think test Id's should be set here, once. `GADMobileAds.sharedInstance()` is singleton so there is no point to set this multiple times.

Reply:
Yeah I know it's a singleton but I did that to keep it consistent with the android side.

Reply:
Does the Android side have a singleton to set `testDevices`? I think it would be nice if the API could initialize testId's globally instead of per ad instance.

What do you think about changing the api to something like this?
```dart
void main() async {
 Admob.initialize(AdService.appId);
 Admob.testDeviceIdentifiers(['abc123', 'cba321']); // <-- Only do this once, globally.
}
```

or maybe `initialize` has optional arg of `testDeviceIdentifiers`

```dart
void main() async {
 Admob.initialize(AdService.appId, testDeviceIdentifiers: ['abc123', 'cba321']);
}
```